### PR TITLE
Community beyond

### PR DESCRIPTION
**Pull request by mdhaber**
Minimal changes.
Other suggestions I can put in this PR if they sound like a good idea:

1. The "Governance" and "Maintainers and contributors" subsections are focused on SciPy itself (internal). "Commmunity beyond the SciPy library" is about relationships with other projects (external), but its sandwiched between the other two internal sections. I would suggest making the order "Governance", "Maintainers and contributors", and "Community beyond the SciPy library".
2. This "Project organization and community" section is all about how the SciPy project works. The "Governance" subsection is about the official leadership roles, like the government of a city. The "Maintainers and contributors" subsection is about the people, like the citizens. The "Community beyond the SciPy library" subsection - as written - focuses on relationships with other projects, like neighboring cities. It's much more specific than the title "Community beyond the SciPy library" currently suggests. I think it would make the subsection - and the section as a whole - feel more complete if the title were more specific. Something like "Reciprocal projects"? "Sister projects"? "Symbiotic projects"? 
3. "then returning to the base SciPy library to suggest and even implement improvements _to enable more effective science applications_" could be removed or made more specific to avoid sounding empty. Is this supposed to evoke the thought that the domain-specific libraries want the code they have developed to be available for use in other domains, and they turn to SciPy to make that possible?

### CODE REVIEW

**Comment by apbard** [comment]
1) +1
2-3) what about something like this? 
```
The scientific Python ecosystem includes many examples
of domain-specific software libraries that are built 
on top of SciPy features. The work done on these libraries
often turns out to be beneficial to the whole ecosystem
being Scipy the catalyst of this cross-fertilisation process. 
For example, there are common contributors to the SciPy and 
Astropy core libraries\cite{astropy-2018}, and what works 
well for one of the codebases, infrastructures, or communities 
is often transferred in some form to the other. At the codebase 
level, the \texttt{binned\_statistic} functionality
is one such cross-project contribution---it was initially
developed in an Astropy-affiliated package
and then placed in SciPy afterwards.
```

File: scipy-1.0/paper.tex [hunk 1]
```diff
@@ -1120,14 +1120,14 @@ \subsection*{Community beyond the SciPy library}
...
-one of the code bases, infrastructures, or communities 
-is often transferred in some form to the other. At the code
-base level, the \texttt{binned\_statistic} functionality
+one of the codebases, infrastructures, or communities 
+is often transferred in some form to the other. At the codebase 
+level, the \texttt{binned\_statistic} functionality
 is one such cross-project contribution---it was initially
 developed in an Astropy-affiliated package
-and then placed in SciPy after the fact.
+and then placed in SciPy afterwards.
```
Comment (mdhaber) [inline scipy-1.0/paper.tex]: avoiding idiomatic phrase

File: scipy-1.0/paper.tex [hunk 2]
```diff
@@ -1120,14 +1120,14 @@ \subsection*{Community beyond the SciPy library}
...
 more effective science applications. For example, there
-are common contributors to the SciPy and astropy core
+are common contributors to the SciPy and Astropy core
 libraries\cite{astropy-2018}, and what works well for 
-one of the code bases, infrastructures, or communities 
-is often transferred in some form to the other. At the code
-base level, the \texttt{binned\_statistic} functionality
+one of the codebases, infrastructures, or communities 
+is often transferred in some form to the other. At the codebase 
+level, the \texttt{binned\_statistic} functionality
```
Comment (mdhaber) [inline scipy-1.0/paper.tex]: "code base" -> "codebase" only for consistency throughout paper (both are valid)

**Comment by mdhaber** [comment]
@apbard Thanks for your thoughts. Could you respond more specifically to 2) and 3)? 
2) Do you agree that there should be a change in subsection title? One of those suggested, or something else?
3) Do you agree with removing "to enable more effective science applications"?
Let's finish this round of edits up, and then would you open a separate PR if you'd like to tweak further?

@WarrenWeckesser We've been at a bit of a standstill with some of these SciPy paper PRs. This is probably the easiest one, so I wondered if you would be willing to take a look and offer your thoughts?

**Comment by WarrenWeckesser** [comment]
The changes look good to me.

Instead of discussing here the other ideas your bring up in the first comment here, just merge this, and put the other proposals in a new pull request.

**Comment by mdhaber** [comment]
@WarrenWeckesser Thanks. I'll do that.

**Pull request merged**